tests/ode_test.py

```diff
@@ -15,7 +15,7 @@
 from functools import partial
 
 from absl.testing import absltest
-import numpy as np
+import numpy as onp
```

Comment:
We're not using this convention any more, and we are trying to be consistent about it. The convention is:
```
import numpy as np
import jax.numpy as jnp
```

Reply:
I can make this tweak!